lib/ansiblelint/rules/PackageHasRetryRule.py

```diff
@@ -76,12 +76,43 @@ class PackageHasRetryRule(AnsibleLintRule):
         "absent",
     ]
 
+    _package_name_keys = [
+        "name",
+        "package",
+        "pkg",
+        "deb",
+    ]
+
+    # attempt to find package name
+    def get_package_name(self, action):
+        for key in self._package_name_keys:
+            found_package_name = action.get(key)
+            if found_package_name:
+                break
```

Comment:
@awcrosby Better return right away. It's safer
```suggestion
                return found_package_name
```